server/src/test/java/keywhiz/KeywhizConfigTest.java

```diff
@@ -48,6 +51,27 @@ public void missingMaximumSecretLengthIsNull() {
     assertNull(config.getMaximumSecretSizeInBytesInclusive());
   }
 
+  @Test
+  public void parseMissingNewSecretOwnershipStrategy() {
+    KeywhizConfig config = loadConfig("new-secret-ownership-strategy-missing.yaml");
+    assertThat(config.getNewSecretOwnershipStrategy()).isEqualTo(
+        KeywhizConfig.NewSecretOwnershipStrategy.NONE);
+  }
```

Comment:
You might want to add "DefaultsToNone" to the end of the test name just document what the expected behavior is.